sprint0/sprint0/Enemies/Stalfos.cs

```diff
@@ -31,11 +31,22 @@ public Stalfos(Texture2D texture, Vector2 location, Game1 game) : base(texture,
                 SpriteEffects.FlipHorizontally
             };
         }
-
+         
         public override void Draw(SpriteBatch spriteBatch)
         {
-            if (damageTimer % 2 == 0)
-                spriteBatch.Draw(Texture, Location, source, Color.White, 0, new Vector2(0, 0), spriteEffects[currentFrame / repeatedFrames], 0);
+            if (frameSpawn >= totalFramesSpawn * repeatedFramesSpawn)
+            {
+                if (damageTimer % 2 == 0)
+                    spriteBatch.Draw(Texture, Location, source, Color.White, 0, new Vector2(0, 0), spriteEffects[currentFrame / repeatedFrames], 0);
+
+            }
+            else
+            {
+                if (frameSpawn < totalFramesSpawn * repeatedFramesSpawn)
+                {
+                    spriteBatch.Draw(game.Content.Load<Texture2D>("Images/Link"), Location, sourcesSpawn[frameSpawn / repeatedFramesSpawn], Color.White);
```

Comment:
can we load this in a factory and store the texture rather than attempting to load it w/every draw?